src/components/structures/MatrixChat.js

```diff
@@ -138,6 +138,19 @@ module.exports = React.createClass({
         return this.props.config.default_is_url || "https://vector.im";
     },
 
+    _getCurrentCreds() {
+        const client = MatrixClientPeg.get();
+
+        return {
+            userId: client.credentials.userId,
+            accessToken: client.getAccessToken(),
+            homeserverUrl: client.getHomeserverUrl(),
+            identityServerUrl: client.getIdentityServerUrl(),
+            guest: client.isGuest(),
+            deviceId: client.getDeviceId(),
+        }
+    },
+
```

Comment:
Any reason for this living in MatrixChat & removing the one in MatrixClientPeg? I recently added the one in MatrixClientPeg to try & get stuff out of MatrixChat (although it wasn't yet used anywhere).